docs/testing.md

```diff
@@ -14,13 +14,13 @@ pip3 install -U "ray[default]"
 ## Start Ray Head Node
 
 ```shell
-ray start --head --node-ip-address=10.0.0.23 --port=6379 --dashboard-host=0.0.0.0
+ ray start --head --dashboard-host 0.0.0.0 --include-dashboard true
 ```
 
-## Start Ray Worker Nodes(s)
+## Start Ray Worker Nodes(s) (Optional, and MacOS doesn't support multi-node cluster)
 
 ```shell
-ray start --address=10.0.0.23:6379 --redis-password='5241590000000000'
+ray start --address=127.0.0.1:6379
```

Comment:
for distributed testing, we wouldn't be able to use `127.0.0.1` here. Perhaps we need separate sections for single node vs distributed?

Reply:
Sure, updated. Thanks for reviewing.